packages/core-api/lib/versions/2/schema/transactions.js

```diff
@@ -3,6 +3,9 @@
 const Joi = require('joi')
 const pagination = require('./pagination')
 
+const container = require('@arkecosystem/core-container')
+const transactionPool = container.resolvePlugin('transactionPool')
```

Comment:
If you don't need the full plugin but only it's options you can use `container.resolveOptions('transactionPool')`.